autoconfigure/pom.xml

```diff
@@ -33,6 +35,7 @@
 
   <modules>
     <module>collector-eventhub</module>
+    <module>storage-applicationinsights</module>
```

Comment:
what's the difference between collector and storage? Should we have Application Insights collector AND storage. So collection part can be used separately from the storage?